cmd/snap/cmd_snap_op_test.go

```diff
@@ -229,6 +229,12 @@ func (s *SnapOpSuite) TestInstallIgnoreRunning(c *check.C) {
 func (s *SnapOpSuite) TestInstallNoPATH(c *check.C) {
 	// PATH restored by test tear down
 	os.Setenv("PATH", "/bin:/usr/bin:/sbin:/usr/sbin")
+	// SUDO_UID env must be unset in this test
+	if sudoUidEnv, isSet := os.LookupEnv("SUDO_UID"); isSet {
+		os.Unsetenv("SUDO_UID")
+		defer os.Setenv("SUDO_UID", sudoUidEnv)
```

Comment:
Nitpick, unconditional unsetting would be fine too, no?

Reply:
Yeah, we could move the unset outside of the if (I think that is what you mean?).